syntax = "proto3";

package adlik.serving;
option cc_enable_arenas = true;

import "adlik_serving/runtime/ml/algorithm/proto/amc_task.proto";
import "adlik_serving/runtime/ml/algorithm/proto/grid_task.proto";

message TaskReq {
  oneof TaskConfig {
    GridTaskReq grid = 1;
    AmcTaskReq amc = 2;
  };
}

message TaskRsp {
  oneof TaskConfig {
    GridTaskRsp grid = 1;
    AmcTaskRsp amc = 2;
  }
}
